cypress/integration/articleFlows/mangageFlows/unlockDiscussion.spec.js

```diff
@@ -0,0 +1,54 @@
+describe('Lock discussion', () => {
+  const getDiscussionLockButton = () =>
+    cy.findByRole('link', {
+      name: /Lock discussion/,
+    });
+
+  const getDiscussionUnlockButton = () =>
+    cy.findByRole('link', {
+      name: /Unlock discussion/,
+    });
+
+  const getDiscussionLockSubmitButton = () => cy.get('.crayons-btn--danger');
+
+  describe('Unlocking the discussion', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/articleEditorV1User.json').as('user');
+
+      cy.get('@user').then((user) => {
+        cy.loginUser(user).then(() => {
+          cy.createArticle({
+            title: 'Test Article',
+            tags: ['beginner', 'ruby', 'go'],
+            content: `This is a test article's contents.`,
+            published: true,
+          }).then((response) => {
+            cy.visit(`${response.body.current_state_path}/manage`);
+            getDiscussionLockButton().click();
+            getDiscussionLockSubmitButton().click();
+          });
+        });
+      });
+    });
+
+    it('should allow a user to unlock a discussion', () => {
+      getDiscussionUnlockButton().click();
+      getDiscussionLockSubmitButton().click();
+
+      getDiscussionLockButton().should('exist');
+    });
```

Comment:
Related to my comment above, but if possible I think it would be great if we could then visit the article page and check that the comment form appears as expected, just to test the desired effect came from the form submission